packages/libs/error-stack/src/serde.rs

```diff
@@ -0,0 +1,153 @@
+//! Implementation of general [`Report`] serialization.
+//!
+//! The value can be of any type, currently only printable attachments and context are supported, in
+//! the near future any values will be supported through the use of hooks.
+//!
+//! The serialized [`Report`] is a list of all current sources with the following output:
+//!
+//! ```json
+//! {
+//!     "context": "context display output",
+//!     "attachments": ["all attachments leading up to this context"],
```

Comment:
Nit, feel free to ignore
```suggestion
//!     "attachments": ["all", "attachments", "leading", "up", "to", "this", "context"],
```